static/app/components/feedback/feedbackItem/feedbackActions.tsx

```diff
@@ -48,11 +49,24 @@ export default function FeedbackActions({
 }
 
 function LargeWidth({feedbackItem}: {feedbackItem: FeedbackIssue}) {
-  const {isResolved, onResolveClick, isSpam, onSpamClick, hasSeen, onMarkAsReadClick} =
-    useFeedbackActions({feedbackItem});
+  const {
+    hasDelete,
+    onDelete,
+    isResolved,
+    onResolveClick,
+    isSpam,
+    onSpamClick,
+    hasSeen,
+    onMarkAsReadClick,
+  } = useFeedbackActions({feedbackItem});
 
   return (
     <Fragment>
+      {hasDelete && (
+        <Button size="xs" onClick={() => openDeleteModal(onDelete)} priority="danger">
+          {t('Delete')}
+        </Button>
+      )}
```

Comment:
Can we show the button in all cases, but make it disabled when the user doesn't have `event:admin` access permissions? We can add a tooltip in that case to explain why it's deleted, and regular users will know that it's possible to delete, and they can ask their admin or whatever instead of emailing us.

Reply:
https://develop.sentry.dev/frontend/design-tenets/#disable-and-explain-dont-hide

Reply:
The idea of a flag so far is that the issue platform isn't ready to GA this. So we'd like to try this for ourselves first. Make sure all is good, then enable for more ppl. Eventually we can delete the feature flag.

Anyone will be able to delete it, not just admin